demo/License/LicenseViewController.swift

```diff
@@ -59,45 +61,16 @@ final class LicenseViewController: UIViewController {
         let label = UILabel()
         label.font = UIFont(name: "AvenirNext-Bold", size: 20)
         label.textColor = .white
-        label.text = L10n.welcomeTitle
+        label.text = L10n.licenseTitle
         return label
     }()
     
-    private let bodyTextView: UITextView = {
```

Comment:
May we use closure-based configuration again since we don't need to parametrize it?

Reply:
done